cartographer/mapping/internal/3d/local_trajectory_builder_3d.cc

```diff
@@ -304,6 +375,23 @@ void LocalTrajectoryBuilder3D::RegisterMetrics(
       "mapping_internal_3d_local_trajectory_builder_latency",
       "Duration from first incoming point cloud in accumulation to local slam "
       "result");
+
+  auto* voxel_filter_fraction = family_factory->NewGaugeFamily(
+      "/mapping/internal/3d/local_trajectory_builder/voxel_filter_fraction",
```

Comment:
See above. Names use underscores and do not contain redundant/unnecessary information, e.g. remove `internal`.

Reply:
Done. Can I remove 'internal' for the others as well for consistency, or would that break something?

Reply:
@spielawa metric names (and labels) cannot contain slashes, we use underscores instead.
https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels

Reply:
@spielawa Changing other code is orthogonal to this PR. See #1237.